AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/TaskController.cs

```diff
@@ -55,6 +58,15 @@ public IActionResult Create(int activityId, TaskEditModel model)
                 ModelState.AddModelError("EndDateTime", "Ending time cannot be earlier than the starting time");
             }
 
+            var activity = _dataAccess.GetActivity(activityId);
+            if ((model.StartDateTime < activity.StartDateTime || model.EndDateTime > activity.EndDateTime) && model.IgnoreWarning == false)

```

Comment:
The model.StartDateTime will come in as the servers timezone. When comparing dates you will need to create a new DateTimeOffset in the Campaign's timezone.  See [EditTaskCommandHandler.cs](https://github.com/HTBox/allReady/blob/master/AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Tasks/EditTaskCommandHandler.cs) for an example of how this is done.